src/components/Dropdown/dropdown.module.scss

```diff
@@ -1,44 +1,59 @@
-$dropdown-hover: #d7d6d6;
+$dropdown-hover: #d0e7ff;
 $white: #fff;
 $theme-blue: #1d1283;
 $theme-active: #87d870;
 $black: #000;
 
 .dropdown {
-    width: 180px;
-    background-color: $white;
-    border: 2px solid $black;
-    border-radius: 0.5rem;
-    color: $black;
+    width: 150px;
+    border-radius: 10px;
+    color: #000;
     position: absolute;
-    top: 58px;
-    right: 16px;
+    top: 70px;
+    right: 20px;
     z-index: 100;
+    display: flex;
+    flex-direction: column;
+    padding: 15px 0;
+    font-size: 0.87rem;
+    justify-content: center;
+    background-color: #fffbfb;
+    box-shadow: rgba(60, 64, 67, 0.3) 0 1px 2px 0,
+        rgba(60, 64, 67, 0.15) 0 2px 6px 2px;
 
     .dropdownList {
         padding: 0;
         margin: 0;
     }
 
     .dropdownItem {
-        padding: 1rem;
-        margin: 0 auto;
+        padding: 15px 30px;
+        /* margin: 0 auto; */
```

Comment:
Do we need this commented code ?

Reply:
Oh no. I ll remove this. Thank you for pointing out